app/models/purchase.py

```diff
@@ -103,7 +104,25 @@ def order_cart_items_from_vendor(vendor_id, date=None):
                             current_user.cart_items)
 
         order = Order(date, vendor_id)
-        # TODO: email vendor
+
+        vendor = User.query.get(vendor_id)
+        merchant_id = current_user.id
+        merchant = Merchant.query.get(merchant_id)
```

Comment:
Makes sense. For some reason, didn't realize that User could be used for both - thanks for catching!